pkg/interfaces/contracts/liquidity-mining/IGaugeAdder.sol

```diff
@@ -21,9 +22,11 @@ import "./ILiquidityGaugeFactory.sol";
 import "./IStakingLiquidityGauge.sol";
 
 interface IGaugeAdder is IAuthentication {
+    // Deprecated. TODO: remove from interfaces, and remove references.
```

Comment:
Nothing; we'll just move it to the checkpointer (done in the next PR on top, #2500).

I'm not doing it here because it breaks the checkpointer, and it sounds out of scope.